lib/Recur/RRuleIterator.php

```diff
@@ -608,6 +614,7 @@ protected function nextYearly()
         // If we got a byDay or getMonthDay filter, we must first expand
         // further.
         if ($this->byDay || $this->byMonthDay) {
+            $occurrence = -1;
```

Comment:
which error is fixed here?

Reply:
`$occurrence` may not be defined by the time it is used on line 654.